pretyping/pretyping.ml

```diff
@@ -419,7 +419,10 @@ let interp_instance ?loc evd l =
          (evd, l :: univs)) (evd, [])
       l
   in
-  if List.exists (fun l -> Univ.Level.is_prop l) l' then
+  let lbound = Global.universes_lbound () in
+  if List.exists (fun l -> Univ.Level.is_prop l &&
+                           Evd.check_leq evd (Univ.Universe.super (Univ.Universe.make l))
+                             (Univ.Universe.make lbound)) l' then
```

Comment:
That looks a bit suspicious. We should still always disallow prop in instances, so I'd come back to the earlier version.

Reply:
Why should we disallow prop in instances?